addons/ai/fnc_taskAttack.sqf

```diff
@@ -31,4 +31,6 @@ params ["_group","_position", ["_radius",0]];
 _group = _group call CBA_fnc_getGroup;
 if !(local _group) exitWith {}; // Don't create waypoints on each machine
 
+[_group] call CBA_fnc_clearWaypoints; // Clear existing waypoints

```

Comment:
I'm a little confused. Is that not what I did at the end with " // Clear existing waypoints" ? Should it be on top instead?